packages/data/README.md

```diff
@@ -749,11 +749,11 @@ const SaleButton = ( { children } ) => {
 
 _Parameters_
 
--   _storeNameOrDescriptor_ `[string|StoreDescriptor]`: Optionally provide the name of the store or its descriptor from which to retrieve action creators. If not provided, the registry.dispatch function is returned instead.
+-   _storeNameOrDescriptor_ `[T]`: Optionally provide the name of the store or its descriptor from which to retrieve action creators. If not provided, the registry.dispatch function is returned instead.
```

Comment:
It would be great to use a more descriptive name instead of `T`. It still won't be perfect as folks won't be able to check all the details of the underlying type but at least that will increase the chance to understand it.